projects/core/directives/hint-controller/hint-controller.directive.ts

```diff
@@ -7,7 +7,7 @@ import {TUI_HINT_CONTROLLER} from './hint-controller.token';
 
 // TODO: v2.0 use in Charts
 @Directive({
-    selector: '[tuiHintContent]',
+    selector: '[tuiHintContent],[tuiHintDirection],[tuiHintMode]',
```

Comment:
@MarsiBarsi please check this too.

Reply:
Hey 👋 It does not seem correct in the current implementation. hintController does not accumulate all the directive from DI above (textfieldController does it, for example). So, if we set `tuiHintDirection` without `tuiHintContent` we customize direction of hint that cannot exist

Reply:
You are right. 
However this implementation doesn't throw any error if someone set hint direction without hint content.
@waterplea do you have any ideas how to provide tuiHintDirection and tuiHintMode through hint controller without this selectors? Should I put them back into input password options token?

Reply:
@KarimovDev is it reasonable for those to be different between tooltip and show/hide hint? How about we still wire those to the controller directive and add an example where you would use empty tuiHintContent="" so that tooltip is not shown, but the directive is matched to pass direction and mode?

Reply:
OK, I've done it in the way above